### DISCUSSION TITLE
Set temperature ?

### DISCUSSION THREAD

**Discussion by litfoo** [Hyundai-Kia-Connect/kia_uvo#834] [General]
Hi there,

Am using this integration for my Kia Sportage 2023 Hybrid Electric.

I have a 'Set Temperature' value that always shows 14°C   but am unsure  what this relates to

Can anyone shed some light please?

Many thanks, litfoo

**Comment by cdnninja** [comment]
Current climate set point. Car reports the lowest setting when not running.

**Reply by litfoo** [reply]
thanks @cdnninja
